hudi-utilities/src/test/resources/delta-streamer-config/source.avsc

```diff
@@ -43,8 +43,41 @@
   }, {
     "name" : "end_lon",
     "type" : "double"
-  },
-  {
+  }, {
+    "name" : "int_val",
```

Comment:
Minor: Instead of naming based on types, can we give relevant names like :distance_in_meters (int_type), seconds_since_epoch(long_val), payment_in_dollars,...

Reply:
Done